django-stubs/test/html.pyi

```diff
@@ -1,12 +1,15 @@
 from collections.abc import Sequence
 from html.parser import HTMLParser
+from re import Pattern
 from typing import Any
 
 from typing_extensions import TypeAlias
 
-WHITESPACE: Any
+ASCII_WHITESPACE: Pattern
```

Comment:
```suggestion
ASCII_WHITESPACE: Pattern[str]
```

Reply:
Thanks, I've updated other `Pattern` which I believe also miss this.

While at it, I also updated the `allowlist.txt` to reference every regex pattern in the `Lazy regex` section. 
I found some missing running `rg ": Pattern" -N | cut -d":" -f2 | xargs -I {} rg {} ./scripts/stubtest/allowlist_todo.txt`